pkg/render/manager.go

```diff
@@ -322,6 +324,11 @@ func (c *managerComponent) managerDeployment() *appsv1.Deployment {
 			Template: *podTemplate,
 		},
 	}
+
+	if overrides := c.cfg.Manager; overrides != nil {
```

Comment:
Shouldn't this be `c.cfg.Manager.ManagerDeployment` ?

Reply:
I think we should probably follow Typha's lead. EgressGateway is a special case because it represents a deployment.

Manager and PolicyRecommendation might theoretically be enhanced in the future to deploy multiple deployments, at which point it would no longer make sense to have it embedded directly into the manager CR.